extensions/amp-story/1.0/story-analytics.js

```diff
@@ -76,6 +75,9 @@ export class StoryAnalyticsService {
 
     /** @const @private {!./variable-service.AmpStoryVariableService} */
     this.variableService_ = getVariableService(win);
+
+    /** @private {!Object} */
+    this.eventsPerPage_ = {};
```

Comment:
nit: Since this is used for lookups, the prototypeless `src/utils/object.js#map` type may be better.

Reply:
That's right, when JS looks up a non-existent reference on an object it has to check the object's prototype chain which includes `Object.prototype`. If a prototypeless object i.e. `const o = Object.create(null)` is used, the lookup stops with properties on the object and it doesn't look in `Object.prototype` too.

Reply:
Got it, thanks!